pkg/arvo/gen/who.hoon

```diff
@@ -0,0 +1,28 @@
+/+  *server
+::  @p request handler
+::
+|=  [[now=@da eny=@ bek=beak] ~ ~]
+|=  [authorized=? =request:http]
+^-  simple-payload:http
+=/  url=(unit pork:eyre)
+  (rush url.request apat:de-purl:html)
+::
+::  url doesn't match expected binding from :launch
+::
+?.  ?=(^ url)
```

Comment:
It's customary to just use `?^` or `?~` for these kinds of typechecks. Same for the check on `t.q.u.url` below.

```suggestion
?~  url
```